Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ruby:3.3
```

Comment:
This is a top-level Dockerfile but only deals w/ ruby.

* move it and `build.sh` to the `ruby-sdk` or
* change it to `Dockerfile.ruby.build` and `build-ruby.sh`